client/components/happychat/style.scss

```diff
@@ -63,7 +63,7 @@
 		display: block;
 		background: lighten( $gray, 30% );
 		bottom: 0;
-		z-index: 10;
+		z-index: 1020;
```

Comment:
<img width="972" alt="screen shot 2016-09-29 at 9 02 26 am" src="https://cloud.githubusercontent.com/assets/1270189/18961917/84a1927e-8623-11e6-9dd7-82fa84f612d0.png">

The rule becomes:
```scss
z-index: z-index( 'root' '.happychat__container.is-open' );
```

And in _z-index.scss we add:
```
'root': (
//...
'.happychat__container.is-open': 1020 //You may want to change this value depending on what you want to be on top of. 
```